src/aof.c

```diff
@@ -1686,23 +1685,23 @@ int loadSingleAppendOnlyFile(char *filename) {
     if (server.aof_load_truncated) {
         serverLog(LL_WARNING,"!!! Warning: short read while loading the AOF file %s!!!", filename);
         serverLog(LL_WARNING,"!!! Truncating the AOF %s at offset %llu !!!",
-            filename, (unsigned long long) valid_up_to);
+        filename, (unsigned long long) valid_up_to);
         if (valid_up_to == -1 || truncate(aof_filepath,valid_up_to) == -1) {
             if (valid_up_to == -1) {
                 serverLog(LL_WARNING,"Last valid command offset is invalid");
             } else {
                 serverLog(LL_WARNING,"Error truncating the AOF file %s: %s",
-                    filename, strerror(errno));
+                filename, strerror(errno));
             }
         } else {
             /* Make sure the AOF file descriptor points to the end of the
-             * file after the truncate call. */
+            * file after the truncate call. */
             if (server.aof_fd != -1 && lseek(server.aof_fd,0,SEEK_END) == -1) {
                 serverLog(LL_WARNING,"Can't seek the end of the AOF file %s: %s",
-                    filename, strerror(errno));
+                filename, strerror(errno));
             } else {
                 serverLog(LL_WARNING,
-                    "AOF %s loaded anyway because aof-load-truncated is enabled", filename);
+                "AOF %s loaded anyway because aof-load-truncated is enabled", filename);
                 ret = AOF_TRUNCATED;
                 goto loaded_ok;
             }
```

Comment:
these are unnecessary changes, and btw function parameters are usually not aligned with the function name.